posts/2022-03-18-BillyDawson_MeetTheTeam.adoc

```diff
@@ -0,0 +1,167 @@
+//
+// Replace AUTHOR_NAME with your name as you'd like it to be displayed, eg: Laura Cowen
+//
+// Example post: 2020-04-02-generate-microprofile-rest-client-code.adoc
+//
+// If adding image into the post add :
+// -------------------------
+// [.img_border_light]
+// image::img/blog/FILE_NAME[IMAGE CAPTION ,width=70%,align="center"]
+// -------------------------
+// "[.img_border_light]" = This adds a faint grey border around the image to make its edges sharper. Use it around screenshots but not           
+// around diagrams. Then double check how it looks.
+// There is also a "[.img_border_dark]" class which tends to work best with screenshots that are taken on dark
+// backgrounds.
+// Change "FILE_NAME" to the name of the image file. Also make sure to put the image into the right folder which is: img/blog
+// change the "IMAGE CAPTION" to a couple words of what the image is
+// // // // // // // //
+
+In this blog series, we’re highlighting some of the amazing individuals who contribute to Open Source software (OSS). We’ll delve into their contributions within OSS, their career journey, how OSS involvement has helped them, and their advice to others for getting involved.
+
+
+== Introduction:
+
+Billy is a cloud software developer within the https://www.ibm.com/docs/en/wasdtfe?topic=SSHR6W/com.ibm.websphere.wdt.doc/topics/welcome_wdt.html[WebSphere Developer Tools] and https://www.ibm.com/docs/en/was-liberty/core?topic=overview-liberty-repository[WebSphere Liberty Repository] team.
+
+Building upon typical team day to day responsibilities managing the Liberty Repository, Billy is the Hursley representative for Developer Advocacy and University Outreach and led the release work for WebSphere Developer Tools for the first half of 2021.
+
+Billy was the focal point for Voice Agent with Watson in the EU cloud and has worked as a DevOps engineer for WebSphere Application Server as a Service. Located at the IBM Hursley laboratory in the UK, Billy is experienced in developing Microservice applications with an affinity for Java and container systems such as Docker and Kubernetes.
+
+Outside of work Billy enjoys gaming, experiencing the outdoors and nature, local community events, motorcycling and cats!
+
+image::/img/blog/Billy.png
+// OR
+image::/img/blog/Billy_Forest.jpg
+// OR image::/img/blog/Billy_IBM.jpeg
+// Unused images: Billy_IBM.jpeg, Billy_BBQ.png, Billy_Dashboard.png, Billy_Slide.png
+
+== Tech introduction:
+
+My primary responsibility is to work on the release and development of our https://marketplace.eclipse.org/content/ibm-liberty-developer-tools[developer tools] for Liberty and WebSphere. The tools encompass plugins for the Eclipse IDE allowing users to develop their applications with Liberty/WebSphere runtimes directly through the IDE. A significant portion of our code base has been open sourced and can be found https://github.com/OpenLiberty/open-liberty-tools[here].
```

Comment:
suggested edit:

 A significant portion of our code base has been open sourced and can be found https://github.com/OpenLiberty/open-liberty-tools[here].
--->
 A significant portion of our code base has been open sourced and can be found in the https://github.com/OpenLiberty/open-liberty-tools[open-liberty-tools GitHub repository].

Reply:
looks like this one and the next couple were missed, prob because they were "hidden"